### ISSUE TITLE
Compile error when using OnMouseDownAsObservable

### ISSUE THREAD

**Issue by mhle** [neuecc/UniRx#189]
When using OnMouseUpAsButtonAsObservable or any other mouse trigger method, the project fails to build on iOS/Android platforms. 

Error CS1061: Type `x' does not contain a definition for `OnMouseUpAsButtonAsObservable' and no extension method `OnMouseUpAsButtonAsObservable' of type `x' could be found. Are you missing an assembly reference? (CS1061) (Assembly-CSharp)

Will there be a version available for touch devices as well?

**Comment by neuecc** [comment]
If use `OnMouse` on touch devices, unity shows warning.
If you want to use OnMouse on touch device, you should make input abstraction layer.

**Comment by mhle** [comment]
yes that's what I'm doing at the moment. But I'd like to have the convenience of OnMouseDrag observable trigger to simulate drag and drop on touch devices. Is it possible to add support and suppress the warnings?

**Comment by oneclickdev** [comment]
so @mhle how did you handle that?

**Comment by oneclickdev** [comment]
@neuecc as I see from here: https://docs.unity3d.com/ScriptReference/Input-simulateMouseWithTouches.html it can work on devices..
moreover, I commented those `//#if !(UNITY_IPHONE || UNITY_ANDROID || UNITY_METRO)` and made a proof build for iPhone and OnMouseDownAsObservable worked just fine. 

so, could you explain in details how to fix it without hardcode or why was it commented, please?
